docs/nodes/validator-node/operator/node-requirements.md

```diff
@@ -18,23 +18,25 @@ To make your validator node and validator fullnode deployment hassle-free, make
 ## Hardware requirements
 
 For running an Aptos **validator node and validator fullnode** we recommend the following hardware resources:
+For running a validator and validator fullnode is that your hardware spec should be good enough to achieve a 30k peak
+TPS on mainnet. In order to evaluate that, you can use the command (https://gist.github.com/igor-aptos/298c87c0a5f095bdae4716b526d82520)
```

Comment:
let's not link a gist potentially could be deleted. we should commit those script in our repo

Reply:
Updated.